openedx/features/content_tagging/rules.py

```diff
@@ -0,0 +1,101 @@
+"""Django rules-based permissions for tagging"""
+
+import openedx_tagging.core.tagging.rules as oel_tagging
+import rules
+from organizations.models import Organization
+
+from common.djangoapps.student.auth import is_content_creator
+
+
+def is_taxonomy_admin(user, taxonomy=None):
```

Comment:
Please add type annotations. For example, does this `taxonomy` parameter accept a `Taxonomy` instance, or only a `ContentTaxonomy` instance? I think it's only the latter.

Reply:
a0e33e1537875dc45145f09ae2b80ab836ff17bd